src/iop/colorchecker.c

```diff
@@ -1362,8 +1351,12 @@ void gui_init(struct dt_iop_module_t *self)
     snprintf(cboxentry, sizeof(cboxentry), _("patch #%d"), k);
     dt_bauhaus_combobox_add(g->combobox_patch, cboxentry);
   }
-  self->request_color_pick = DT_REQUEST_COLORPICK_OFF;
-  dt_bauhaus_widget_set_quad_paint(g->combobox_patch, dtgtk_cairo_paint_colorpicker, CPF_NONE, NULL);
+
+  GtkWidget *hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, DT_BAUHAUS_SPACE);
```

Comment:
These four lines can be simplified to 
`dt_color_picker_new(self, DT_COLOR_PICKER_POINT_AREA, g->combobox_patch);`
with the addition of 
`g_signal_connect(G_OBJECT(g->combobox_patch), "quad-pressed", G_CALLBACK(_color_picker_callback), self);`
if the picker_cst needs to be set, but as mentioned above this currently would cause an issue.

Also, I think the quad picker looks better aligned than the stand-alone color picker. They used to look very similar if I remember correctly, but the toggle button version has had more space added around it which now causes the combo to somewhat misalign.

Also remove g->colorpicker (in dt_iop_colorchecker_gui_data_t) if you make the change.